packages/primitives/src/index.js

```diff
@@ -8,6 +8,11 @@ export const Note = 'NOTE';
 export const Path = 'PATH';
 export const Rect = 'RECT';
 export const Line = 'LINE';
+export const FormField = 'FORM_FIELD';
```

Comment:
Not a strong opinion, but would `Form` be more semantic? For `Field` I imagine like an input, but this is actually a form wrapper

Reply:
Hi @diegomura,

thanks for your response. Let's see what I suggested here in the past: https://github.com/diegomura/react-pdf/pull/2845#discussion_r1721040015. 🤔 

My initial thought was to align the names with the ones used in `pdfkit` ([here](https://github.com/foliojs/pdfkit/blob/831179e0674076a7b7775991620f26065967b250/docs/forms.md#form-annotation-methods)). So `TextInput` becomes `FormText`, `Picker` becomes `FormPicker`, and so on. But you made a valid point [back in 2022](https://github.com/diegomura/react-pdf/pull/2013/files#r1014982991) suggesting we should stick to the native web primitives. I got used to the current names while preparing the PR.

Regarding `FormField`: I chose the same name as `pdfkit` mainly to align our (react-pdf) and their [(pdfkit) docs](https://github.com/foliojs/pdfkit/blob/831179e0674076a7b7775991620f26065967b250/docs/forms.md#field-names) and keep them similar. I don't know if that's practical.

Reply:
What about something like `FormGroup`? Form rather sounds like the single and complete set of inputs, but they rather are a subsection. E.g., you could separate billing address and shipping address into two `groups`.

Reply:
Hi @PhilippBloss,

your suggestion makes sense to me, considering what pdfkit states in their docs as well:

> Using the formField method you might create a shipping field that is added to the root of the document, an address field that refers to the shipping field as it's parent, and a street Form Annotation that would refer to the address field as it's parent.

What do you think of `fieldset`(<https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset>):

> The <fieldset> HTML element is used to group several controls as well as labels (<label>) within a web form.

Reply:
Sounds good as well. Would give the decision to @diegomura

Reply:
Would take over the docs, except @natterstefan has already started with those

Reply:
> Would take over the docs, except @natterstefan has already started with those

Hi @PhilippBloss, I haven't taken care of the docs yet. You can take over the docs if you want to or we can share the task. It's fine for me.

Also thanks for the review. I will try to apply your suggestions tomorrow or at least in the next few days.

Thanks for the ping @diegomura.